src/connectors/menu/connectMenu.ts

```diff
@@ -65,7 +66,7 @@ export type MenuConnectorParams = {
    *
    * If a facetOrdering is set in the index settings, it is used when sortBy isn't passed
    */
-  sortBy?: SortBy<MenuItem>;
+  sortBy?: SortBy<SearchResults.HierarchicalFacet>;
```

Comment:
This seems odd to me that this connector would receive a hierarchical facet shape, but that's what the received shape most looks like (see provided examples in description).

Reply:
Yes, it's a hierarchical facet, as it's the only facet we have with "menu"/ one selection behaviour we have